### DISCUSSION TITLE
Peek into sub circuits while running.

### DISCUSSION THREAD

**Discussion by tylergajewski** [hneemann/Digital#1208] [Q&A]
I am not sure if I missed this feature but in logisim I'm able to peek into my sub circuits to see their current state with each clock pulse.  I did not see if that is possible here.  For example, when debugging my RAM circuit i can peak into it to see what each register is doing,

**Comment by hneemann** [comment]
see #84

**Reply by tylergajewski** [reply]
oh sorry thank you very much
